wcfsetup/install/files/lib/data/language/LanguageEditor.class.php

```diff
@@ -141,25 +139,32 @@ public function export($packageIDArray = array(), $exportCustomValues = false) {
 		
 		// sort categories
 		ksort($items);
-		
+
+		// write language file
+		$filename = FileUtil::getTemporaryFilename($this->languageCode, '.xml');
+
+		$xml = new XMLWriter();
+		$xml->beginDocument('language', 'http://www.woltlab.com', 'http://www.woltlab.com/XSD/maelstrom/language.xsd', array('languagecode' => $this->languageCode, 'languagename' => $this->languageName, 'countrycode' => $this->countryCode));
+
 		foreach ($items as $category => $categoryItems) {
 			// sort items
 			ksort($categoryItems);
-			
-			// category header
-			echo "\t<category name=\"".$category."\">\n";
+
+			$xml->startElement('category', array('name' => $category));
```

Comment:
How are your guidelines for the short array notation?

`[]` or `[ ]`
 `['foo']` or `[  'foo' ]` 

 Because actually there is no unique notation visible.